services/exercises/project/db/domain/nodes/user.py

```diff
@@ -1,7 +1,7 @@
 from project.db.domain.entity import Entity
 from project.db.domain import date_fmt_str
 from datetime import datetime
-from typing import Dict, Union
+from typing import Dict, Union, Optional
 
 
 class User(Entity):
```

Comment:
Is this supposed to duplicate users? I don't think this needs to exists in the graph db. I think the users_ids from the auth service should be stored on the textbook

Reply:
hmm that is a better idea I think. I will change the model appropriately